package.json

```diff
@@ -72,6 +73,7 @@
     "redux": "^3.x.x",
     "redux-immutable": "3.0.8",
     "redux-logger": "*",
+    "remarkable": "^1.7.1",
```

Comment:
Since we're using Remarkable directly, do we need `react-remarkable` anymore?

Reply:
Nope, I think we can remove it. I also want to try removing `ReactMarkdown` from the OAS3 component and replace it with `dangerouslySetInnerHTML` like in the 2.0 component.